Cargo.lock

```diff
@@ -973,21 +984,6 @@ dependencies = [
  "winapi",
 ]
 
-[[package]]
-name = "clap"
```

Comment:
Nice! You'll probably want to update the **deny.toml** to no longer skip the `clap` tree _(and `structopt-derive`!)_